core/include/webview/detail/engine_base.hh

```diff
@@ -318,6 +320,19 @@ protected:
     }
   }
 
+  void dispatch_size_default(bool m_owns_window) {
+    if (!m_owns_window || m_script_is_init) {
+      return;
```

Comment:
`m_script_is_init` guards against Windows dispatching `set_size` prematurely in the execution of `add_init_script`. This is not relevant to other platforms, which do not empty the event queue during bind.